benchmarks/graphics_pipeline/main.cpp

```diff
@@ -701,49 +454,296 @@ void ProjApp::Setup()
         grfx::ShaderModuleCreateInfo shaderCreateInfo = {static_cast<uint32_t>(bytecode.size()), bytecode.data()};
         PPX_CHECKED_CALL(GetDevice()->CreateShaderModule(&shaderCreateInfo, &mPsShaders[j]));
     }
+}
 
-    CreateSpherePipelines();
+void ProjApp::SetupFullscreenQuadsResources()
```

Comment:
Sounds like it is just shaders being created? Maybe `CreateFullscreenQuadsShaders()`?

Reply:
Changed